### PR TITLE
Use TimeDuration helper for NullDuration types

### PR DESCRIPTION
**Pull request by 9a24f0**
This PR is an attempt to resolve GH-2203.
Any kind of feedback are welcome 🎉

### CODE REVIEW

**Comment by FloorD** [comment]
Hi @9a24f0 👋 we would love to send you a little k6 swag pack as per "thank you" for contributing to k6 this hacktoberfest. Please send your address details to floor[at]grafana.com and I'll take care it's shipped asap.

File: lib/types/types.go [hunk 1]
```diff
@@ -230,7 +230,7 @@ func (d NullDuration) ValueOrZero() Duration {
 
 // TimeDuration returns a NullDuration's value as a stdlib Duration.
 func (d NullDuration) TimeDuration() time.Duration {
-	return time.Duration(d.Duration)
+	return d.TimeDuration()
```
Comment (MStoykov) [inline lib/types/types.go:233]: This should be reverted as this is just calling the same function, effectively making an endless recursion

**Review by MStoykov — CHANGES_REQUESTED** [review]

File: lib/types/types.go (hunk 1 above)
Comment (9a24f0) [reply lib/types/types.go]: Thank you so much! This is reverted in 047c231

**Review by codebien — APPROVED** [review]
LGTM, thanks

**Review by MStoykov — APPROVED** [review]
LGTM, thanks for making the changes @9a24f0 :bow:

**Review by na-- — APPROVED** [review]
Thanks!

**Pull request merged**